src/20250120_Multi_FebruaryFundingUpdate/FebruaryFundingUpdate_20250120.s.sol

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {GovV3Helpers, IPayloadsControllerCore, PayloadsControllerUtils} from 'aave-helpers/src/GovV3Helpers.sol';
+import {GovernanceV3Ethereum} from 'aave-address-book/GovernanceV3Ethereum.sol';
+import {EthereumScript, PolygonScript, OptimismScript, ArbitrumScript, AvalancheScript} from 'solidity-utils/contracts/utils/ScriptUtils.sol';
+import {AaveV3Avalanche_FebruaryFundingUpdate_20250120} from './AaveV3Avalanche_FebruaryFundingUpdate_20250120.sol';
+import {AaveV3Ethereum_FebruaryFundingUpdate_20250120} from './AaveV3Ethereum_FebruaryFundingUpdate_20250120.sol';
+import {AaveV3Polygon_FebruaryFundingUpdate_20250120} from './AaveV3Polygon_FebruaryFundingUpdate_20250120.sol';
+import {AaveV3Optimism_FebruaryFundingUpdate_20250120} from './AaveV3Optimism_FebruaryFundingUpdate_20250120.sol';
+
+/**
+ * @dev Deploy Ethereum
+ * deploy-command: make deploy-ledger contract=src/20250120_Multi_FebruaryFundingUpdate/FebruaryFundingUpdate_20250120.s.sol:DeployEthereum chain=mainnet
+ * verify-command: FOUNDRY_PROFILE=mainnet npx catapulta-verify -b broadcast/FebruaryFundingUpdate_20250120.s.sol/1/run-latest.json
+ */
+contract DeployEthereum is EthereumScript {
+  function run() external broadcast {
+    // deploy payloads
+    address payload0 = GovV3Helpers.deployDeterministic(
```

Comment:
I think this is not needed here (as its the avalanche payload)

Reply:
wdym?